mail/templates/invalid_format_error/body.html

```diff
@@ -0,0 +1,13 @@
+{% extends "email_base.html" %}
+{% block content %}
+  <tr>
+      <td style="background-color: #ffffff;">
+        <p>We’re sorry, there was a problem processing your video</p>
+        <p>Video: <a href={{ video_url }}>{{video_title}}</a></p>
```

Comment:
Can we put quote marks around the `{{  }}` here and the other `href` attributes in this PR?